src/Illuminate/Foundation/Testing/TestCase.php

```diff
@@ -117,7 +117,7 @@ protected function setUpTraits()
     {
         $uses = array_flip(class_uses_recursive(static::class));
 
-        if (isset($uses[RefreshDatabase::class])) {
+        if (isset($uses[RefreshDatabase::class]) || isset($uses[RefreshDatabaseLazily::class])) {
```

Comment:
Is this extra check actually needed?
As it uses `class_uses_recursive` it should find the `RefreshDatabase` trait and execute the statement.